nomad/resources_endpoint.go

```diff
@@ -47,8 +47,7 @@ func getMatches(iter memdb.ResultIterator) ([]string, bool) {
 		case *structs.Node:
 			id = raw.(*structs.Node).ID
 		default:
-			//s.logger.Printf("[ERR] nomad: unexpected type for resources context; not a job, allocation, node, or evaluation")
-			// TODO
+			r.srv.logger.Printf("[ERR] nomad.resources: unexpected type for resources context; %T \n", t)
```

Comment:
Can you change `;` to `:` just for consistency and you also don't need the `\n`. The logger adds it automatically.